libr/core/cmd_debug.c

```diff
@@ -2128,7 +2128,7 @@ static void r_core_cmd_bp(RCore *core, const char *input) {
 						   if (p) {
 							   do {
 								   ++p;
-							   } while ( isdigit (*p) );
+							   } while ( !IS_WHITESPACE(*p) );
```

Comment:
this will overflow because you shuold end when there's a \0 at the end.. and this is exactly what strchr does. there's no need to test for tabs or so. just use strchr and make things simpler :)